### DISCUSSION TITLE
Shelly plus 1pm not flashing shelly homekit

### DISCUSSION THREAD

**Discussion by a-veek** [mongoose-os-apps/shelly-homekit#1587] [Q&A]
hello together,

i convert my Shelly plus 1PM to the latest stock firmware, that is now 1.44.
i tried to flash shelly homekit and it does not work.
what can i do?
Thanks

Greetings

**Comment by markirb** [comment]
Can you try the version linked here:

https://github.com/mongoose-os-apps/shelly-homekit/issues/1567

**Reply by a-veek** [reply]
hmm, does not work.
tried to update over shelly web and OTA direkt.
still shelly firmware.

**Reply by Salle88** [reply]
Reboot shelly, try again.
Maybe factory reset -> Update to Stock 1.4.4 After that Update from this file.

**Reply by a-veek** [reply]
no, sorry, nothing works.
the device is on Stock 1.4.4.
can i grab a log?
